src/app/components/Labels/LabelSortableItem.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { SortableElement } from 'react-sortable-hoc';
+import { debounce, noop } from 'proton-shared/lib/helpers/function';
+import { Icon, Button } from 'react-components';
+
+import ToggleNotify from './ToggleNotify';
+import EditLabelButton from './EditLabelButton';
+import DeleteLabelButton from './DeleteLabelButton';
+
+function LabelItem({ label, onEditLabel, onRemoveLabel }) {
+    const { ID, Name, Color, Exclusive, Notify } = label;
+
+    return (
+        <tr style={{ backgroundColor: 'white', cursor: 'move' }}>
+            <td>
+                <Icon name="text-justify" />
+            </td>
+            <td>
+                {Exclusive === 1 && <Icon name="folder" style={{ fill: Color }} className="icon-16p mr1" alt={Name} />}
```

Comment:
There is only 2 cases for Exclusive, 1 or 0, so you can use `? : `.

Reply:
Easier to read^^